server.go

```diff
@@ -133,10 +133,7 @@ func (s *Server) NewClient(ctx context.Context, namespace string) (client.Client
 // Note that the HostPort and ConnectionOptions fields of client.Options will always be overridden.
 func (s *Server) NewClientWithOptions(ctx context.Context, options client.Options) (client.Client, error) {
 	options.HostPort = s.frontendHostPort
-	options.ConnectionOptions = client.ConnectionOptions{
-		DisableHealthCheck: false,
-		HealthCheckTimeout: timeoutFromContext(ctx, time.Minute),
-	}
+	options.ConnectionOptions = client.ConnectionOptions{}
```

Comment:
FWIW, this isn't necessary (it'll be an empty struct anyways)

Reply:
thanks for pointing this out.  just removed the unused connection options.